tox.ini

```diff
@@ -138,12 +134,13 @@ commands =
 description = Python type checking with mypy
 basepython = {[testenv:py3]basepython}
 deps =
-  mypy>=1.10.0,<2.0
+  mypy>=1.10.0,<1.14
```

Comment:
Since these are developer tools, only used in CI, it would be safe to use `==` to pick a version and then let dependabot or renovate update the version with a PR. That would keep you up to date, but only allow updates that pass the tests.

Reply:
Removed and introduced in the `instructlab-sdg` package. See SDG release [v0.6.2](https://github.com/instructlab/sdg/releases/tag/v0.6.2).